less/index.less

```diff
@@ -57,3 +57,12 @@ html, body {
     height: 100%;
 }
 
+header nav{
+	min-height: 400px;
+	min-width: 400px;
+	background: #6b6b6b;
+	padding: 40px 20px;
+	border-radius: 5px;
+	box-shadow: 5px 55px 50px -20px #994646;
+}
```

Comment:
way more Less needs to be added, were missing Mixins and variables here as well as general styling of the page.